app/models/pglogical_subscription.rb

```diff
@@ -188,6 +189,15 @@ def create_subscription
     self
   end
 
+  def assert_schemas_match!
```

Comment:
With this method name, I was expecting an `local_result == remote_result` check somewhere.  But we're really validating specific things in each schema against an expected schema.

Maybe `assert_valid_schemas!` would be a more accurate name.